scripts/test.ts

```diff
@@ -13,6 +13,7 @@ import {
 } from '@angular/platform-browser-dynamic/testing';
 
 import './matchers';
+import { environment } from '../demo/src/environments/environment.qa';
```

Comment:
in theory ng cli should replace `import { environment } from './environments/environment';`
with current env file